tv_apps/dashboard/style/dashboard.css

```diff
@@ -140,3 +140,33 @@ iframe {
   width: 100%;
   height: 100%;
 }
+
+/* Intro animation */
+.active #bottom-panel {
+  transform-origin: 50vw calc(16rem - 50vh);
```

Comment:
It's pretty hard to understand the value 16rem - 50vh. Would you mind to write more comments to explain why we use (16rem - 50vh)